test/e2e/pkg/manifest.go

```diff
@@ -115,6 +115,15 @@ type ManifestNode struct {
 	// pause:      temporarily pauses (freezes) the node
 	// restart:    restarts the node, shutting it down with SIGTERM
 	Perturb []string `toml:"perturb"`
+
+	// Misbehaviors sets how a validator behaves during consensus at a
+	// certain height. Multiple misbehaviors at different heights can be used
+	//
+	// An example of a misbehavior
+	//    "double-prevote,10"
```

Comment:
This example is no longer valid. We should have a list of all misbehaviors as well.

Reply:
There is a list in maverick/consensus/behaviors.go that I can reference to